tests/requirements-relaxed.txt

```diff
@@ -0,0 +1,161 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile --allow-unsafe --output-file=tests/requirements-relaxed.txt --strip-extras tests/requirements-relaxed.in
```

Comment:
It depends on the CWD, usually. And with Dependabot, that may also change. Ideally, this would be in nox with pip-compile arguments to produce a custom header with a custom command stated here.

Reply:
This is fine. It's simpler than changing directories and then running `pip-compile`.

Reply:
sure, but I think it should be consistent with the existing `requirements.txt` file right? https://github.com/ansible/ansible-documentation/blob/677f51223473b8eb509171fe351c38969e416ec1/tests/requirements.txt#L5

Reply:
The noxfile in https://github.com/ansible/ansible-documentation/pull/258 is set up to run from the repo root, so this will be changed in the next dependency update PR.